core/src/main/java/software/amazon/awssdk/auth/Aws4Signer.java

```diff
@@ -301,14 +306,13 @@ private String createCanonicalRequest(SdkHttpRequest request,
     private String createStringToSign(String canonicalRequest,
                                       Aws4SignerRequestParams signerParams) {
 
-        final String stringToSign = new StringBuilder(signerParams.getSigningAlgorithm())
-                .append(SignerConstants.LINE_SEPARATOR)
-                .append(signerParams.getFormattedSigningDateTime())
-                .append(SignerConstants.LINE_SEPARATOR)
-                .append(signerParams.getScope())
-                .append(SignerConstants.LINE_SEPARATOR)
-                .append(BinaryUtils.toHex(hash(canonicalRequest)))
-                .toString();
+        final String stringToSign = signerParams.getSigningAlgorithm() +
```

Comment:
It's functionally identical. It read cleaner without it to me, and cleaned up some "unnecessary string builder" warnings that some IDEs give.